### PR TITLE
Remove unused view field from Analysis class

### PR DESCRIPTION
**Pull request by elharo**
## Description

Remove unused view field from Analysis class

## Motivation and Context
One less warning

## Impact
none

## Test Plan
CI

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [x] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [x] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [x] Adequate tests were added if applicable.
- [x] CI passed.

## Release Notes

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by elharo** [comment]
2024-02-27T23:33:05.1670158Z [ERROR] com.facebook.presto.hive.TestHiveDistributedQueriesWithThriftRpc.testViewAccessControlInvokerDefault  Time elapsed: 0.038 s  <<< FAILURE!
2024-02-27T23:33:05.1672167Z java.lang.RuntimeException: View already exists: 'tpch.test_view_access'

**Comment by elharo** [comment]
Completely different failure this time, so yes, it's just flaky:

2024-03-04T14:16:17.7263256Z [ERROR] Tests run: 473, Failures: 1, Errors: 0, Skipped: 472, Time elapsed: 20.789 s <<< FAILURE! - in com.facebook.presto.mongodb.TestMongoDistributedQueries
2024-03-04T14:16:17.7267200Z [ERROR] com.facebook.presto.mongodb.TestMongoDistributedQueries.init  Time elapsed: 20.106 s  <<< FAILURE!
2024-03-04T14:16:17.7268776Z java.lang.RuntimeException: Prematurely reached end of stream

**Comment by elharo** [comment]
The most important thing to note here is not the change itself, which is minor at most, but that I had to rerun the CI tests at least three times before it got to a passing state. This is not a good place to be, and the team should think about how we can improve this.

**Comment by mlyublena** [comment]
nit: add a bit more context to description, for example: "Remove unused view field from Analysis class"

**Review by mlyublena — DISMISSED** [review]

**Comment by elharo** [comment]
@rschlussel @ajaygeorge for approval

**Review by tdcmeehan — APPROVED** [review]

**Comment by rschlussel** [comment]
> Tangential, but this flakiness is probably fixable. It looks like two different tests create a view with the same name, so there's a race condition

i have a fix for that that i added to a different pr when i had the same issue.  just waiting for final approval to merge https://github.com/prestodb/presto/pull/22011/commits.

**Review by rschlussel — APPROVED** [review]

**Review by tdcmeehan — CHANGES_REQUESTED** [review]
Looks good but please squash commits

**Comment by elharo** [comment]
Commits are squashed

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**
